src/backend/parser/cypher_clause.c

```diff
@@ -1371,6 +1536,33 @@ static Query *transform_cypher_match_pattern(cypher_parsestate *cpstate,
     return query;
 }
 
+/*
+ * Function to make a target list from an RTE. Borrowed from AgensGraph and PG
+ */
+static List *makeTargetListFromJoin(ParseState *pstate, RangeTblEntry *rte)
```

Comment:
please change to make_target_list_from_join

Reply:
OK Thanks